### ISSUE TITLE
Solving Token Issue Between Frontend and Backend

### ISSUE THREAD

**Issue by zudiay** [bounswe/2021SpringGroup5#611]
We cannot connect the login required routes on backend with frontend. Frontend adds the access token sent at login to default headers, but the backend also requires the csrf token and session-id to be added to the header.

**Comment by zudiay** [comment]
The csrf token and session id are sent from backend on login response headers, they are visible in developer tools of the browser. But when I tried to console.log the response headers, those fields are lost.  After a long research, I learned that those fields cannot be read due to some cross origin restrictions. Here are the references: [1](https://stackoverflow.com/questions/44107665/how-to-access-a-browser-cookie-in-a-react-app) [2](https://docs.djangoproject.com/en/4.0/ref/csrf/)

**Comment by zudiay** [comment]
I requested the help of backend and frontend teams.

**Comment by atasoyfurkan** [comment]
- I made a meeting with the backend team to help Didem. After two hours of trying to find what caused the problem, I proposed to change the token functionalities of the backend because that one was incompatible with React.
- The backend team change the token functionalities but the problem couldn't be solved. After debugging, I proposed to change the authentication header to Authorization which is a default name for the header.
